src/test/tests.ts

```diff
@@ -896,7 +893,15 @@ describe('Server', function () {
       //do nothing
     });
 
-    client.unsubscribe(subId);
+    setTimeout(() => {
+      // TODO
+      // Hagaico I had the need to add this timeout here
```

Comment:
Before debuging i want to understand current behavior.
Without setTimeout is onOperation called? You can just add console log to it